### PR TITLE
Ms small design fixes

### PR DESCRIPTION
**Pull request by msecret**
Tracks #306. See individual commit messages for more information.

### CODE REVIEW

**Comment by msecret** [comment]
The acceptance tests are failing in travis due to a change in login configuration at a different level of cloud.gov. These tests will be turned off in a separate PR.

File: static_src/components/users.jsx [hunk 1]
```diff
@@ -159,6 +159,11 @@ export default class Users extends React.Component {
                 onRemovePermissions={ this.handleRemovePermissions }
             />
           </div>
+          <p>For more information on different roles and what they mean see <a
```
Comment (jeremiak) [inline static_src/components/users.jsx]: @msecret what do you think about putting this at the top of the page for now? I think its worthwhile context and its hard to find if there are a decent number of users in an org.

**Comment by jeremiak** [comment]
["Remove user from org"](https://github.com/18F/cg-deck/blob/staging-alpha/static_src/components/user_list.jsx#L69) action  should by styled red since it is a delete action

**Comment by jeremiak** [comment]
The ["title bar"](https://github.com/18F/cg-deck/blob/ms-small_design_fixes/static_src/app.jsx#L66-L74) needs to be removed to satisfy "Remove the page title bar (with the dark background color) from the secondary header"

**Comment by jeremiak** [comment]
Looks great to me, nice work.

**Pull request merged**